packages/web-components/src/components/notice-choice/notice-choice.ts

```diff
@@ -299,6 +316,69 @@ class NoticeChoice extends StableSelectorMixin(LitElement) {
     return `${ddsPrefix}--notice-choice`;
   }
   static styles = styles; // `styles` here is a `CSSResult` generated by custom WebPack loader
+  checkBoxLegalChange($event: any) {
+    const legalCheckbox = $event.target;
+    const isChecked = legalCheckbox.checked ? true : false;
+    const legalTextError = legalCheckbox.parentNode.querySelector('.nc-error');
+    legalCheckbox.value = isChecked ? 1 : 0;
+    legalTextError.style.display = isChecked ? 'none' : '';
```

Comment:
@deathcave if hide-error-message is true, in that case, legalTextError is null, you cannot access style.display.
add an additional check here.

Reply:
Fixed Please check